src/hotspot/cpu/x86/stubGenerator_x86_64_arraycopy.cpp

```diff
@@ -727,6 +732,7 @@ address StubGenerator::generate_disjoint_copy_avx3_masked(address* entry, const
   __ ret(0);
 
   if (MaxVectorSize == 64) {
+    UnsafeCopyMemoryMark ucmm(this, !is_oop && !aligned, false, ucme_exit_pc);
```

Comment:
This is not related to Unsafe::setMemory?

Reply:
No.  Reviewing the code I saw this as a potential error, as `arraycopy_avx3_large` could cause a SIGBUS which wouldn't be caught.  It conforms to the other instances of copy in the code.  I think it was missed by the original developer.